src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.cmp

```diff
@@ -338,76 +339,103 @@
+                                                    {!v.batchMetadata.labels.batchProcessSizeLabel}
+                                                </label>
+                                                <lightning:helptext content="{!$Label.c.bgeBatchBatchProcessSizeHelp}" />
+                                                <lightning:input aura:id="batchProcessSize"
+                                                                label="{!v.batchMetadata.labels.batchProcessSizeLabel}"
+                                                                type="number" step="0" required="true" variant="label-hidden"
+                                                                value="{!v.batchInfo.batchProcessSize}"/>
+                                            </lightning:layoutItem>
+                                            <lightning:layoutItem class="slds-p-around_x-small" size="12">
+                                                <lightning:input aura:id="runOpportunityRollupsWhileProcessing"
+                                                                label="{!v.batchMetadata.labels.runOpportunityRollupsWhileProcessingLabel}"
+                                                                type="checkbox" variant="label-hidden" class="inputCheckbox"
+                                                                checked="{!v.batchInfo.runOpportunityRollupsWhileProcessing}"/>
+                                                <label class="slds-form-element__label" for="runOpportunityRollupsWhileProcessing">
+                                                    {!v.batchMetadata.labels.runOpportunityRollupsWhileProcessingLabel}
+                                                </label>
+                                                <lightning:helptext content="{!$Label.c.bgeBatchRunOpportunityRollupsWhileProcessingHelp}" />
+                                            </lightning:layoutItem>
+                                            <lightning:layoutItem class="slds-p-around_x-small" size="12">
+                                                <lightning:input aura:id="processUsingScheduledJob"
+                                                                label="{!v.batchMetadata.labels.processUsingScheduledJobLabel}"
+                                                                type="checkbox" variant="label-hidden" class="inputCheckbox"
+                                                                checked="{!v.batchInfo.processUsingScheduledJob}"/>
+                                                <label class="slds-form-element__label" for="processUsingScheduledJob">
+                                                    {!v.batchMetadata.labels.processUsingScheduledJobLabel}
+                                                </label>
+                                                <lightning:helptext content="{!$Label.c.bgeBatchProcessUsingScheduledJobHelp}" />
```

Comment:
Re: previous comment, this helptext can/should be removed too: bgeBatchProcessUsingScheduledJobHelp